libs/tradesman/lib/widgets/notification_card_widget.dart

```diff
@@ -6,82 +7,79 @@ import 'package:redux_comp/app_state.dart';
-    Key? key,
-    required this.titleText,
-    required this.date,
-  }) : super(key: key);
+  final Store<AppState> store;
+  
+  const NotificationCardWidget(
+      {Key? key,
+      required this.titleText,
+      required this.descText,
+      required this.date,
+      required this.store,})
+      : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return InkWell(
-      onTap: () => {},
-      child: Card(
-        margin: const EdgeInsets.fromLTRB(12, 8, 12, 8),
-        color: Theme.of(context).primaryColorLight,
-        shape: RoundedRectangleBorder(
-          borderRadius: BorderRadius.circular(20.0),
-        ),
-        elevation: 4,
-        child: Padding(
-          padding: const EdgeInsets.all(5.0),
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.start,
-            children: [
-              Row(
-                mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                children: [
-                  Padding(
-                    padding: const EdgeInsets.only(left: 15),
-                    child: Padding(
-                      padding: const EdgeInsets.all(5.0),
-                      child: Text(titleText,
-                          style: const TextStyle(
-                              fontSize: 20,
-                              color: Colors.white,
-                              fontWeight: FontWeight.bold)),
-                    ),
-                  ),
-                  Padding(
-                    padding: const EdgeInsets.only(right: 15),
-                    child: Text(date,
-                        style: const TextStyle(
-                            fontSize: 18, color: Colors.white70)),
-                  ),
-                ],
-              ),
-              Row(
-                children: [
-                  const Padding(padding: EdgeInsets.only(left: 15)),
-                  SizedBox(
-                    // wi: 55,
-                    width: MediaQuery.of(context).size.width / 1.3,
-                    child: Padding(
-                      padding: const EdgeInsets.only(top: 8.0),
-                      //A bid was accepted
-                      //A bid was submitted
-                      //A new bid was made on your job
-                      child: SingleChildScrollView(
-                        child: Text(
-                          titleText == "Accepted!"
-                              ? "Your bid was accepted. A chat has been created between you and your client."
-                              : titleText == "Submitted!"
-                                  ? "Your bid was submitted."
-                                  : "A bid was made on your advert.",
-                          style: const TextStyle(
-                              fontSize: 20, color: Colors.white),
-                          maxLines: 3,
-                          overflow: TextOverflow.ellipsis,
+    return StoreProvider<AppState>(
```

Comment:
So I think I removed logout action from the view model here cause it didn't make much sense but then I had merge conflicts with develop and I am afraid I may have overwrote your file @caaaramia

Reply:
scratch that I removed logout action from tradesman listings, I have no idea what happened here, I think its from develop and it's fine...

Reply:
all appears to be fine